pkg/daemon/ceph/osd/daemon.go

```diff
@@ -501,7 +501,7 @@ func getActiveDirs(currentDirList []string, savedDirMap map[string]int, configDi
 
 // releaseLVMDevice deactivates the LV to release the device.
 func releaseLVMDevice(context *clusterd.Context, volumeGroupName string) error {
-	if op, err := context.Executor.ExecuteCommandWithCombinedOutput(false, "", "lvchange", "-anvv", volumeGroupName); err != nil {
```

Comment:
why do verbose args cause failures? 🤔

Reply:
PR description is updated with the error... The intention was to set verbose arguments, but they are just invalid.